frontend/src/metabase-lib/Question.ts

```diff
@@ -1174,7 +1185,7 @@ class QuestionInner {
       return (
         q &&
         new Question(q.card(), this.metadata())
-          .setParameters([])
+          .setParameters(getTemplateTagParametersFromCard(q.card()))
```

Comment:
This method is used to compare whether 2 questions are considered equal. We cannot drop parameters as before because there are some new settings that are saved only in `parameters` and not `template-tags`. But we still want to ignore parameters that were auto-generated during the drill-down process (when coming from a dashboard etc). So instead of dropping all parameters we leave parameters that can be computed from the card itself.